models2/Preferences.es6.js

```diff
@@ -0,0 +1,118 @@
+import RedditModel from './RedditModel';
+const T = RedditModel.Types;
+
+export default class PreferencesModel extends RedditModel {
+  static type = 'preferences';
+
+  static PROPERTIES = {
+    affiliateLinks: T.bool,
+    allowClicktracking: T.bool,
+    beta: T.bool,
+    clickgadget: T.bool,
+    collapseReadMessages: T.bool,
+    compress: T.bool,
+    credditAutorenew: T.bool,
+    defaultCommentSort: T.string, // It would be nice to have T.oneOf like react here
```

Comment:
we could easily add it. something to consider though, what should we see here if the value doesn't validate? an empty string, or a null value?